README.md

```diff
@@ -5,79 +5,86 @@ OpenShift Application Platform
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 [![Jenkins](https://ci.openshift.redhat.com/jenkins/buildStatus/icon?job=devenv_ami)](https://ci.openshift.redhat.com/jenkins/job/devenv_ami/)
 
-OpenShift Origin is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin adds developer and operations-centric tools on top of Kubernetes to enable rapid application development, easy deployment and scaling, and long-term lifecycle maintenance for small and large teams.
+***OpenShift Origin*** is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin enables teams of all sizes to quickly develop, deploy, and scale applications, reducing the length of development cycles and operational effort.
 
-**Features:**
+Features
+--------
+
+**For developers:**
+
+* Easily build applications with integrated service discovery and persistent storage
+* Quickly and easily scale applications to handle periods of increased demand
+  * Support for automatic high availability, load-balancing, health checking, and fail-over
+* Push source code to your Git repository and automatically deploy containerized applications
+* Web console and command-line client for building and monitoring applications
+
+**For system administrators:**
 
-* Easily build applications with integrated service discovery, DNS, load balancing, failover, health checking, persistent storage, and fast scaling
-* Push source code to your Git repository and automatically roll out containerized applications (micro or macro)
```

Comment:
I've been pondering this one, and I'm not sure if it's worth mentioning. I think it's useful that we can run both stateful and stateless apps, but I'm not sure it's valuable to explicitly call out microservice/monolithic applications as the distinction isn't that important. For microservices, you need service discovery (to find your other pieces), and for most older apps, you generally need persistent storage (though that could be true of microservices too - consider a store like etcd or mysql)